unicodetools/src/main/java/org/unicode/xml/GeneratePropertyValues.java

```diff
@@ -0,0 +1,1749 @@
+package org.unicode.xml;
+
+import com.ibm.icu.dev.tool.UOption;
+import com.ibm.icu.util.VersionInfo;
+import java.io.*;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.unicode.props.PropertyParsingInfo;
+import org.unicode.props.UcdProperty;
+import org.unicode.props.UcdPropertyValues.*;
+
+public class GeneratePropertyValues {
```

Comment:
Another very large file... :-/
Could you please add a `/** ... */` comment to each top-level Java class here (one per .java file) to say, in broad strokes, what each class does and how it connects with the other top-level classes?

Reply:
Done